mkdocs.yml

```diff
@@ -0,0 +1,54 @@
+site_name: Djinni
+site_description: >-
+  Djinni is a tool for generating cross-language type declarations and interface bindings. 
+  It's designed to connect C++ with either Java or Objective-C.
+copyright: Copyright &copy; 2020 cross-language-cpp
+repo_url: https://github.com/jothepro/djinni-documentation
+markdown_extensions:
+  - pymdownx.tabbed
+  - pymdownx.highlight
+  - pymdownx.superfences
+  - pymdownx.details
+  - admonition
+  - plantuml_markdown:
+      format: svg
+  - toc:
+      permalink: true
+  - pymdownx.emoji:
+      emoji_index: !!python/name:materialx.emoji.twemoji
+      emoji_generator: !!python/name:materialx.emoji.to_svg
+plugins:
+  - monorepo
+  - search
+  - git-revision-date-localized
+extra:
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/cross-langugage-cpp
+    - icon: fontawesome/brands/slack
+      link: https://mobilecpp.herokuapp.com/
+theme:
+  name: material
+  custom_dir: overrides
+  logo: assets/logo.svg
+  features:
+    - navigation.tabs
+    - navigation.instant
+  icon:
+    repo: fontawesome/brands/github
+extra_css:
+  - stylesheets/theme.css
+nav:
+  - Home: index.md
+  - Getting Started: 
+    - overview.md
+    - setup.md
+    - example.md
+    - contributing.md
+  - Generator: '!include ./djinni-generator/mkdocs.yml'
```

Comment:
This is how external (submodule) repositories can be included inside the website. Each repository has it's own `mkdocs.yml` configuration with a `nav`-tree and the `monorepo` plugin will merge them into one.